src/main/java/com/questhelper/QuestHelperPlugin.java

```diff
@@ -307,7 +312,7 @@ protected void startUp() throws IOException
 
 		final BufferedImage icon = Icon.QUEST_ICON.getImage();
 
-		panel = new QuestHelperPanel(this);
+		panel = new QuestHelperPanel(this, skillIconManager);
```

Comment:
nit: Since `skillIconManager` is unused atm, it might be best to remove this requirement for now